lambdas/backend-client/src/routing-config-api-client.ts

```diff
@@ -81,4 +125,38 @@ export const routingConfigurationApiClient = {
 
     return { ...data };
   },
+
+  async update(
+    token: string,
+    id: RoutingConfig['id'],
+    routingConfig: RoutingConfig
+  ): Promise<Result<RoutingConfig>> {
+    if (!isValidUuid(id)) {
```

Comment:
Why is this validation needed here? If it's an invalid id, wouldn't the API just return a 404?